deployment/cre/jobs/settings.go

```diff
@@ -46,17 +46,35 @@ func VerifyCRESettings(s string) error {
 		}
 	}
 	for id, org := range data.Org {
-		if err := ensureSchema(org); err != nil {
+		// TODO to be enforced after upgrading deployed nodes and settings
+		//if strings.HasPrefix(id, "org_") {
+		//	errs = errors.Join(errs, fmt.Errorf("invalid org id %s: must not be prefixed org_", id))
+		//} else
```

Comment:
We have to allow org prefix (and configure duplicate settings) until we are only running the new code.